doc/developer/debugging.rst

```diff
@@ -53,10 +57,8 @@ For better security, the session cookie is accessible only via http protocol (me
 and is secure (meaning, the cookie is transmitted only in https requests, not in http requests).
 For this reason, you should have your application running on https also in your development environment.
 
-To achieve that, if it is not already done, copy the file ``docker-compose.override.sample.yaml``
-to ``docker-compose.override.yaml``.
-
-Then access the application on `https://localhost:8484/ <https://localhost:8484/>`_.
+With the default configuration in the file ``docker-compose.override.yaml`` your application will be
```

Comment:
With the default configuration in the file ``docker-compose.override.yaml``, your application will be